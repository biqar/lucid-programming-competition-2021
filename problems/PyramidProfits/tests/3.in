10
7e6daf Jude 80
34c75f 0576db 60
0576db Jude 20
6d66b3 34c75f 35
19904e b107af 45
fbcf09 7e6daf 95
b107af 7e6daf 45
bd29a6 e1fc7c 50
e1fc7c 0576db 40
237e24 7e6daf 20
20
b107af 530800
237e24 1249600
0576db 1801100
237e24 697500
237e24 1762700
fbcf09 726500
237e24 1241300
237e24 1364100
6d66b3 982200
bd29a6 428900
6d66b3 689100
0576db 1927300
34c75f 1628100
7e6daf 1397200
b107af 528200
237e24 321300
e1fc7c 1943500
b107af 1771300
b107af 175300
19904e 1175800
