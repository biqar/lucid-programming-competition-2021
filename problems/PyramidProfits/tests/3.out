0576db 1294320
19904e 190480
237e24 1061840
34c75f 325733
6d66b3 70195
7e6daf 7233996
Jude 10108480
b107af 1314824
bd29a6 17156
e1fc7c 172636
fbcf09 552140
